swarm/src/connection/pool.rs

```diff
@@ -798,6 +808,7 @@ where
                         handler,
                         endpoint,
                         abort_notifier: _,
+                        creation: _,//Ignoring the time it took for the connection to _fail_
```

Comment:
```suggestion
                        creation: _,// Ignoring the time it took for the connection to _fail_.
```

Reply:
Might be something worth exposing eventually as well. That said, fine to skip for now.